File: CodingBat/Java/Logic-2/closeFar.java
public boolean closeFar(int a, int b, int c) {
  return Math.abs(b - a) <= 1 && Math.abs(c - a) >= 2 && Math.abs(c - b) >= 2
		  || Math.abs(c - a) <= 1 && Math.abs(b - a) >= 2 && Math.abs(b - c) >= 2;
}

File: CodingBat/Python/Logic-2/close_far.py
def close_far(a, b, c):
  return (abs(b - a) >= 2 and abs(b - c) >= 2 and abs(c - a) <= 1) or (abs(c - a) >= 2 and abs(b - c) >= 2 and abs(b - a) <= 1)
